doc/architecture/performance/generator.rst

```diff
@@ -0,0 +1,70 @@
+Generator
+=========
+
+Overview
+--------
+
+This component is responsible for generating requests and storing them in a format 
+that could be submitted to the server. The user can declare the requests by leveraging
+the functions inside the library in ``CCF/tests/perf-system/generator/generator.py``.
+The user can generate requests from the library by either calling the command line tool 
+in ``CCF/tests/perf-system/generator/generate_packages.py`` or by creating a script 
+calling the functions of the library, such as the ``CCF/tests/perf-system/generator/loggin_generator.py`` 

```

Comment:
Sorry for the late nit pick but can you call this file `logging_generator` instead of `loggin_generator`?

Reply:
Should I change the name of the file in this PR too?

Reply:
I made the changes for the `loggin_generator` to `logging_generator` in the `.rst` file. Although, the file in the CCF repo is still called `loggin_generator` so the link from the `.rst` file is broken. Is it ok if I fix the name file in this PR, or should I create a new PR to fix this? (Maybe the same PR with the perf-system name change to Piccolo)

Reply:
Maybe mark both this threads as resolved and create a file renaming PR which we'll bypass branch protection to merge. This is a known issue https://github.com/microsoft/CCF/issues/4395